French

el00ruobuob opened the pull request:
This is the French Localization of Monerujo.

It includes the new folder along with the three files fully translated.

I may have missed a few typos as usual, and it may includes incoherencies, especially regarding the infinitive / conjugated forms of verbs in the strings that were sometimes hard to guess from the string name or existing translations.
So i believe a few corrections should be made after seeing this "live" on my smartphone.

@erciccione i let you ping our french speakers?

el00ruobuob committed:
French Language creation

el00ruobuob committed:
Complete French Translation.

- about.xml completed;
- strings.xml completed;
- help.xml completed.

m2049r commented on app/src/main/res/values-fr/strings.xml:
```diff
@@ -0,0 +1,375 @@
+<resources>
+    <string name="app_name" translatable="false">monerujo</string>
+    <string name="login_activity_name" translatable="false">monerujo</string>
```
please delete all strings with ```translatable="false"``` - they may only be in the original files.

m2049r replied:
also, i think you may be working off an old version. please rebase to current master.

el00ruobuob replied:
ok, done.

m2049r commented:
@el00ruobuob sorry - i meant those strings should not be translated and completely removed from the translations.

el00ruobuob committed:
Correction after reviews.

- Removed translation=false strings.

el00ruobuob commented:
Remove like this?
It's also pulled and rebase on latest.

m2049r commented:
> Remove like this?

perfect, thanks!

the following strings from ```strings.xml``` are not translated (probably because they were not in the master branch when you pulled it first):
```XML
    <string name="menu_changepw">Change Passphrase</string>
    <string name="changepw_progress">Change Password in progress</string>
    <string name="changepw_failed">Change Password failed!</string>
    <string name="changepw_success">Password changed</string>
    <string name="prompt_changepw">New Passphrase for %1$s</string>
    <string name="prompt_changepwB">Repeat Passphrase for %1$s</string>
    <string name="generate_bad_passwordB">Passphrases do not match</string>
    <string name="generate_empty_passwordB">Passphrase may not be empty</string>
    <string name="generate_crazypass_label">Wallet Files Restore Password</string>
```

also there is a new a chapter "Wallet Files Recovery Password" in ```help_details``` of ```help.xml``` which is new.

el00ruobuob commented:
Ok, i'm doing them.
Btw, i wonder how to translate CrAzYpass, or if the term should be translated at all.

el00ruobuob committed:
Translation of newly added strings and help topics related to CrAzYpass #234

el00ruobuob commented:
@m2049r Enjoy!

erciccione commented:
Pinging @assylias @glv2 @cryptochangements34

m2049r commented:
@el00ruobuob merci beaucoup!

yes, CrAzYpass is not to be translated :)

m2049r commented:
heh - please remove:

- ```info_send_xmrto_success_status_label```
- ```info_send_xmrto_success_status```

from ```strings.xml```. Also ```'``` need to be escaped as ```\'```.

m2049r commented on app/src/main/res/values-fr/help.xml:
```diff
@@ -0,0 +1,247 @@
+        vos fonds. Il est très imprtant de la garder sécurisée et confidentielle, car elle donne <em>à quiconque</em>
+        le contrôle total de vos Moneroj ! Si vous ne l’avez pas écrite à un emplacement sécurisé, faites-le !
+        <h2>Mdp de restauration des fichiers du Portefeuille</h2>
+        Assurez vous de noter ce mot de passe. Si vous réinitialisez votre appareil ou désinstallez l'application,
```
pls excape the ```'``` - why is the apostrophe a different one than e.g. "n’importe" or "l’accès"?

el00ruobuob replied:
Cause on french keyboard we use to take a single quote as an apostrophe... I just corected that, removed the to xmrto strings mention above, and escaped everything in strings.xml

el00ruobuob committed:
corrections after review

- Strings of xmr.to succes_status removed;
- quotes escaped in strings.xml;
- quote remplaced by apostrophe in help.

cryptochangements34 commented on app/src/main/res/values-fr/help.xml:
```diff
@@ -0,0 +1,247 @@
+    ]]></string>
+
+    <string name="help_create_view"><![CDATA[
+        <h1>Créer un Portefeuille - Audit</h1>
```
maybe make this "Portefeuille d'Audit" to match the GUI ?

el00ruobuob replied:
sure.

cryptochangements34 commented on app/src/main/res/values-fr/help.xml:
```diff
@@ -0,0 +1,247 @@
+    <string name="help_wallet"><![CDATA[
+        <h1>Le Portefeuille</h1>
+        <h2>Analyse</h2>
+        Comme Monero aime garder les choses confidentielles, nous devoins scanner la chaîne de blocs
```
did you mean "nous devons" ?

el00ruobuob replied:
i think so.

el00ruobuob committed:
corrections after review

French typos & enhancements from @cryptochangements34

cryptochangements34 commented on app/src/main/res/values-fr/help.xml:
```diff
@@ -0,0 +1,247 @@
+        <p>Veuillez noter que cette clef secrète est seulement valable 24h après le lancement de la
+        transaction !</p>
+        <h2>Compte à Rebours XMR.TO !</h2>
+        <p>Une fois qu le compte à rebours atteint zéro, vous devrez obtenir un nouveau devis depuis
```
Une fois **que**

el00ruobuob replied:
done

cryptochangements34 commented on app/src/main/res/values-fr/help.xml:
```diff
@@ -0,0 +1,247 @@
+        <p>Veuillez noter que cette clef secrète est seulement valable 24h après le lancement de la
+        transaction !</p>
+        <h2>Compte à Rebours XMR.TO !</h2>
+        <p>Une fois qu le compte à rebours atteint zéro, vous devrez obtenir un nouveau devis depuis
```
une fois *que*

el00ruobuob replied:
damned!

cryptochangements34 commented on app/src/main/res/values-fr/strings.xml:
```diff
@@ -0,0 +1,366 @@
+    <string name="prompt_mainnet">Sélection Réseau</string>
+    <string name="connect_stagenet">StageNet</string>
+    <string name="connect_testnet">TestNet</string>
+    <string name="connect_mainnet">MainNet</string>
```
shouldn't it just be "Stagenet", "Testnet", and "Mainnet"? That's the way it is done in the official CLI/GUI

el00ruobuob replied:
I kept this unchanged. global change @m2049r ?

el00ruobuob replied:
I can lowercase the net part in every language if you're ok.

m2049r replied:
i'm very ok with that . thanks @el00ruobuob

m2049r replied:
hmm - unless they are the same in every language, in which case we should mark them untranslatable and remove them from the translations and fix only the original?

el00ruobuob replied:
IMO they should be the same in all languages. I'll do the change in values/strings.xml, add the untranslatable to it, and remove the three strings from other strings.xml. (after a rebase for norvegian)

cryptochangements34 commented on app/src/main/res/values-fr/strings.xml:
```diff
@@ -0,0 +1,366 @@
+
+    <string name="text_retry">Touchez pour réessayer</string>
+    <string name="text_noretry_monero">Maintenant on est coincé ici !</string>
+    <string name="text_noretry">Oh-oh, XMR.TO n\'a pas laire disponible pour le moment !</string>
```
" l'air " not " laire "

el00ruobuob replied:
done. don't know where it came from...

el00ruobuob committed:
Changed Mainnet/Testnet/Stagenet to untranslatable and removed strings from localizations.

m2049r reviewed (approved):
thank you for your patience!

m2049r commented:
@cryptochangements34 let me know if it's fine with you too

erciccione commented:
Hey @el00ruobuob we got some new strings after #244 got merged. Can you rebase and translate the new strings as well?

el00ruobuob committed:
Rebased onto latest, and integration of #254 & #244 in French.

el00ruobuob commented:
@erciccione @m2049r rebased it is.
I did translate those new strings from #244 & #254 also.
